coalib/output/ConsoleInteraction.py

```diff
@@ -316,7 +316,7 @@ def print_affected_files(console_printer,
                          file_dict,
                          color=True):
```

Comment:
Shortlog: tense is not correct, should be Imperative, aka: Correct typos
also, shouldn't be a space before the colon.